sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -309,14 +313,17 @@ public interface AnnotatedQueriesUserRepositoryCodeSnippet extends CosmosReposit
 
 ```java
 public interface AnnotatedQueriesUserReactiveRepositoryCodeSnippet extends ReactiveCosmosRepository<User, String> {
-    @Query(value = "select * from c where c.firstName = @firstName and c.lastName = @lastName")
+    @Query("select * from c where c.firstName = @firstName and c.lastName = @lastName")
```

Comment:
Same here, please update the readme where these 2 code snippets are mentioned, and update their line numbers and code snippet in the readme as well.